src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/TopicController.cs

```diff
@@ -32,5 +34,21 @@ public async Task<ActionResult<Topic>> GetTopicAsync(Guid topicId)
 
             return Ok(topic);
         }
+
+        // POST api/theme/{themeId}/topics
+        [HttpPost("theme/{themeId}/topics")]
+        public async Task<ActionResult<TopicViewModel>> CreateTopicAsync(
+            CreateTopicViewModel topic, Guid themeId)
+        {
+           topic.ThemeId = themeId;
+           var result = await _topicService.CreateTopicAsync(topic);
```

Comment:
I would change this so that CreateTopicAsync takes 2 parameters, a themeId and a createTopicRequest, rather than set the ThemeId on the Request as it comes into this method - it's a bit neater that way and with no surprising side effects - just a general rule of thumb when it comes to development as a whole :)